containerd-shim-spin/src/engine.rs

```diff
@@ -290,6 +346,52 @@ impl Engine for SpinEngine {
     fn can_handle(&self, _ctx: &impl RuntimeContext) -> Result<()> {
         Ok(())
     }
+
+    fn supported_layers_types() -> &'static [&'static str] {
+        &[
+            "application/vnd.wasm.content.layer.v1+wasm",
```

Comment:
nit: consider layer media types names as consts rather than literals and replaces usages.

Reply:
Those changes are coming from #16, so I wouldn't block this given these are unrelated changes. 

https://github.com/spinkube/containerd-shim-spin/commit/7b1e07a99a6d1a2882609ddc66eed337565be90e contains the changes in this PR.

Reply:
Great point for #16!

Reply:
@devigned this should be resolved and ready for another review